File: backend/src/main/java/com/backend/petplace/domain/mypage/dto/MyPageUserPets.java
package com.backend.petplace.domain.mypage.dto;

import com.backend.petplace.domain.pet.entity.Gender;
import java.time.LocalDate;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class MyPageUserPets {

  private final Long id;
  private final String name;
  private final Gender gender;
  private final LocalDate birthDate;
  private final String type;

  public MyPageUserPets(Long id, String name, Gender gender, LocalDate birthDate, String type) {
    this.id = id;
    this.name = name;
    this.gender = gender;
    this.birthDate = birthDate;
    this.type = type;
  }

}

File: backend/src/main/java/com/backend/petplace/domain/mypage/dto/MyPageUserPets.kt
package com.backend.petplace.domain.mypage.dto

import com.backend.petplace.domain.pet.entity.Gender
import java.time.LocalDate

data class MyPageUserPets(
    val id: Long,
    val name: String,
    val gender: Gender,
    val birthDate: LocalDate?,
    val type: String?
)
